src/library/dlgcoverartfullsize.cpp

```diff
@@ -219,7 +224,8 @@ void DlgCoverArtFullSize::slotCoverInfoSelected(
 }
 
 void DlgCoverArtFullSize::mousePressEvent(QMouseEvent* event) {
-    if (!m_pCoverMenu->isVisible() && event->button() == Qt::LeftButton) {
+    if (m_pCoverMenu != nullptr && !m_pCoverMenu->isVisible() &&
+            event->button() == Qt::LeftButton) {
```

Comment:
This means if there is no menu instance the cover window can not be dragged anymore.
How about this:
```suggestion
    if (event->button() != Qt::LeftButton) {
        return;
    }
    if ((m_pCoverMenu != nullptr && !m_pCoverMenu->isVisible()) ||
            m_pCoverMenu == nullptr) {
```

Reply:
I didn't notice that, I tested it before the requested changes, when there was no menu label, left click wasn't working probably and wasn't able to turn off the full size. With this change now it can. Thank you! :)